packages/components/src/select-control/styles/select-control-styles.ts

```diff
@@ -75,7 +81,7 @@ export const Select = styled.select`
 		${ fontSizeStyles };
 		${ sizeStyles };
 
-		${ rtl( { paddingLeft: 8, paddingRight: 24 } )() }
+		${ rtl( { paddingLeft: 8, paddingRight: 24 } ) }
```

Comment:
I was thinking again about this change, and was wondering — if `rtl` [returns a function](https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/utils/rtl.js#L71-L81), why are we removing the `()` and therefore not executing that function?

I'm probably missing some aspects of how styling with Emotion works.

Reply:
Becuase emotion executes interpolated functions on render, for example if you have a props callback... this is the same thing as the other mixins, we're just not reading the props.